.github/actions/bump-version/core.js

```diff
@@ -0,0 +1,79 @@
+// Copied these commands out of the github actions toolkit
+// because actually depending on @actions/core requires me to check
+// in node_modules and 34MB of dependencies, which I don't want to do.
```

Comment:
You should be able to run npm install as part of the workflow to avoid checking it in

Reply:
That said, I can understand why they would want to encourage checking in node_modules for shared actions even though it seems gross to js people. Checking in allows actions to run predictably and consistently every time without things going wrong due to npm.org being offline or loose versioning (e.g. with >= instead of exact pinning) leading to unexpected changes of behavior in a future install. My issue really wasn't with committing dependencies as with the absolutely huge amount of dependencies being pulled in when all I wanted was to read and write a few environment variables.